### PR TITLE
Calico logo, note in StatefulSets -- Test scaling and Batch -- simple workflow

### PR DESCRIPTION
**Pull request by nikipat**
*Description of changes:* 1. changed image in calico/_index.md to official project Calico logo
2. added a note about duration of pods getting launched in statefulsets/testscaling.md

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by nikipat** [comment]
Added another correction in batch/simple workflow (commit 5f627f66aa6219d0ccb9bb5fee081ae481554489)

File: content/batch/workflow-simple.md [hunk 1]
```diff
@@ -47,11 +47,12 @@ Duration:            3 seconds
 STEP               PODNAME         DURATION  MESSAGE
  ✔ whalesay-2kfxb  whalesay-2kfxb  2s        
 ```
+Make a note of the workflow's name from your output.
 
-Confirm the output.
+Confirm the output by running the following command:
```
Comment (buzzsurfr) [inline content/batch/workflow-simple.md]: If we sub out the workflow name, then we should also state to substitute the name out...

“Confirm the output by running the following command, substituting `<workflow name>` for the name of the workflow above.”

Comment (nikipat) [reply content/batch/workflow-simple.md]: Yes you are right. My text is actually:
"Make a note of the workflow's name from your output.

Confirm the output by running the following command:

```bash
argo logs <workflow name>
```"
But your sentence is much clearer. Will make changes and commit

Comment (dalbhanj) [reply content/batch/workflow-simple.md]: looks like this is fixed

**Pull request merged**